tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -107,7 +107,7 @@ function main() {
   mkdir -p ${TMPDIR}/third_party
   pushd ${RUNFILES%org_tensorflow}
   for header in $(find protobuf -name \*.h); do
-    cp --parents "$header" ${TMPDIR}/google;
+    rsync -R "$header" ${TMPDIR}/google;
```

Comment:
Thanks for offering the fix. 
This script is used on multiple platforms. I'm a little worried that rsync may be missing from some of them. Can we do the safe thing here, namely something like
DEST_DIR="${TMPDIR}/google/$(dirname ${header})"
mkdir -p "${DEST_DIR}"
cp "${header}" "${DEST_DIR}/"